frontend/assets/javascripts/src/modules/toggle.js

```diff
@@ -10,12 +10,14 @@
 define(['$', 'bean'], function ($, bean) {
 
     var TOGGLE_BTN_SELECTOR = '.js-toggle',
-        TOGGLE_DATA_ELM     = 'toggle',
-        TOGGLE_DATA_LABEL   = 'toggle-label',
-        TOGGLE_CLASS        = 'is-toggled',
-        ELEMENTS_TO_TOGGLE  = '[data-toggle-hidden]';
+        TOGGLE_LABEL_SELECTOR = '.js-toggle-label',
```

Comment:
Booo, I love lining things up for no reason at all.

(this is fine, ignore me)